src/utils/hex.ts

```diff
@@ -0,0 +1,49 @@
+// Eth
+import { bigIntToHex, bytesToHex } from "../deps.ts";
+import { PrefixedHexString, stripHexPrefix } from "../deps.ts";
+
+/**
+ * @param hex - A decimal string.
+ */
+export function toHexString(decimal: string): PrefixedHexString {
+  return bigIntToHex(BigInt(decimal));
+}
+
+/**
+ * @param hex - A hex string.
+ * @param length - The final length in bytes of the hex string.
+ */
+export function padString(
```

Comment:
can be one function no? 

function pad(value: bigint | string | ...) and then inside, we perform if checks for type?

Reply:
or even, have all these functions as local functions (not exported)

and export pad() that switches on the type of the value, and calls these util functions

Reply:
(nitpicking, but just to expose 1 function for all the codebase?)

Reply:
mmmmmm I don't know, I based it off what ethereumjs-monorepo was doing where they have `intToHex`, `bytesToHex`, `bigIntToHex`. Can make a single function if you prefer yes